test/auto_parallel/hybrid_strategy/CMakeLists.txt

```diff
@@ -100,3 +100,10 @@ if((WITH_GPU) AND (LINUX))
   set_tests_properties(test_semi_auto_parallel_llama_model_pir
                        PROPERTIES TIMEOUT "180" LABELS "RUN_TYPE=HYBRID")
 endif()
+if((WITH_GPU) AND (LINUX))
+  py_test_modules(
+    test_pir_reshard_nd_mesh_func MODULES test_pir_reshard_nd_mesh_func ENVS
+    "http_proxy=;https_proxy=;PYTHONPATH=../..:${PADDLE_BINARY_DIR}/python")
+  set_tests_properties(test_pir_reshard_nd_mesh_func
+                       PROPERTIES TIMEOUT "35" LABELS "RUN_TYPE=HYBRID")
```

Comment:
timeout 35 might be to short,  it is better to setting timeout as 90 for 4 gpus  unitests take into account the overhead of communicators initialization.

Reply:
In local test it costs about 10s, it can be updated when adding more cases in the future.